test/plugins/collections.test.js

```diff
@@ -20,6 +23,31 @@ const getCollectionMock = (collection) => {
     return mock;
 };
 
+const getCollectionsMock = (collections) => {
+    if (Array.isArray(collections)) {
+        return collections.map(getCollectionMock);
+    }
+
+    return getCollectionMock(collections);
+};
+
+// Get the mock pipeline in testPipelines using the input id
+const getPipelineMockFromId = (id) => {
+    let result = null;
+
+    testPipelines.forEach((pipeline) => {
+        if (pipeline.id === id) {
+            result = hoek.clone(pipeline);
+
+            result.update = sinon.stub();
+            result.toJson = sinon.stub().returns(pipeline);
+            result.remove = sinon.stub();
+        }
```

Comment:
The reason I kept the `return` outside was to have 1 common `return` statement to account for both the null case as well as in the case that the pipeline is found. I'll see if there's a way I can better refactor this. Let me know if you know of a better way